Src/FluentAssertions/Primitives/DateTimeAssertions.cs

```diff
@@ -928,6 +928,36 @@ public AndConstraint<TAssertions> BeIn(DateTimeKind expectedKind, string because
         return new AndConstraint<TAssertions>((TAssertions)this);
     }
 
+    /// <summary>
+    /// Asserts that the <see cref="DateTime"/> does not represent a value in the <paramref name="unexpectedKind"/>.
+    /// </summary>
+    /// <param name="unexpectedKind">
+    /// The unexpected <see cref="DateTimeKind"/> that the current value must not represent.
+    /// </param>
```

Comment:
🤔 This reads like a double negative. Maybe `unexpectedKind` should just be named `kind`

Reply:
Hmm I followed the summary of the positive `BeIn()`.

Do you mean the summary or the the param-part?

Reply:
Both ;-)